src/config.c

```diff
@@ -248,26 +283,34 @@ dictType optionToLineDictType = {
 };
 
 dictType optionSetDictType = {
-    dictSdsCaseHash,            /* hash function */
+    distCStrCaseHash,           /* hash function */
     NULL,                       /* key dup */
     NULL,                       /* val dup */
-    dictSdsKeyCaseCompare,      /* key compare */
+    dictCStrKeyCaseCompare,     /* key compare */
     dictSdsDestructor,          /* key destructor */
     NULL,                       /* val destructor */
     NULL                        /* allow to expand */
 };
 
-dictType sdsKeyValueDictType = {
-    dictSdsCaseHash,            /* hash function */
+dictType configSetDictType = {
+    distCStrCaseHash,           /* hash function */
     NULL,                       /* key dup */
     NULL,                       /* val dup */
-    dictSdsKeyCaseCompare,      /* key compare */
+    dictCStrKeyCaseCompare,     /* key compare */
     dictSdsDestructor,          /* key destructor */
-    dictSdsDestructor,          /* val destructor */
+    configValuesDestructor,     /* val destructor */
     NULL                        /* allow to expand */
 };
 
-typedef struct standardConfig standardConfig;
+dictType KeyValueDictType = {
```

Comment:
The name here needs to be something like `ptrCStrDictType`. `keyValueDictType` is too generic.